Makefile

```diff
@@ -102,43 +102,11 @@ clean-npm: ## Removes local containers, images, volumes, for app application (n=
 # Databas Utilities
 ##############################################################################
 
-db-install-cli: ## Install EF CLI
```

Comment:
Removed any commands that require local installation of .NET6 SDK.  They still exist in the `/api/Makefile` as that would work within a Development Container.